lib/pure/os.nim

```diff
@@ -764,19 +1012,23 @@ when not defined(windows) and not weirdTarget:
     else: result = S_ISLNK(rawInfo.st_mode)
 
 const
-  ExeExts* = when defined(windows): ["exe", "cmd", "bat"] else: [""] ## \
-    ## platform specific file extension for executables. On Windows
-    ## ``["exe", "cmd", "bat"]``, on Posix ``[""]``.
+  ExeExts* = ## Platform specific file extension for executables.
```

Comment:
No, the syntax shall be ``const name = value ## doc comment``.

Reply:
Doing
```nim
const
  ExeExts* = when defined(windows): ["exe", "cmd", "bat"] else: [""] ## Platform specific (...)
```
makes doc comment disappear from the docs.

Reply:
This might be related: https://github.com/nim-lang/Nim/issues/9899

Reply:
Ok, will change it later then.